data/map.txt

```diff
@@ -26752,6 +26779,26 @@ planet Starcross
 		threshold 2500
 		fleet "Small Militia" 22
 
+planet "Starting Rubin"
+	attributes kimek "coalition station" shipping wealthy
+	landscape land/space3
+	description `Starting Rubin was far from being the first station constructed around the Kimek homeworld, but it is the oldest one to still be inhabited. It was built as an orbital equivalent to the hunger towers found on the planet it orbits, as a safety measure against the disasters that may affect Ki Patek Ka.`
+	description `Since the residents need few of the food produced, it soonly became the station's primary export, transforming it in one of the largest trading hubs of the region. It has also the largest population of every Coalition station, capable of supporting tens of thousands of Kimek.`
+	spaceport `Despite only having the population of a small city, the arrivals deck of Starting Rubin are as crowded as the largest spaceports. Civilian ships, belonging to all the three Coalition species, can be seen arriving or departing every couple minutes.`
```

Comment:
`the arrivals deck of Starting Rubin are as crowded` -> `the arrivals deck of Starting Rubin is as crowded` (are -> is)

Reply:
Changed